src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,17 @@
+import NextAuth from 'next-auth'
+import Providers from 'next-auth/providers'
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+export default NextAuth({
+  // Configure one or more authentication providers
+  providers: [
+    Providers.Discord({
+      clientId: `${process.env.DISCORD_CLIENT_ID}`,
+      clientSecret: `${process.env.DISCORD_CLIENT_SECRET}`,
+    }),
+  ],
+
+  secret: 'QKHPhLFyCSABKvYz',
```

Comment:
Just double checking if you need to commit the above :)

Reply:
@sotir-ls Thanks for the finding. I've used a random hash for it as a placeholder. It should be used from env